templates/base.html

```diff
@@ -13,7 +13,7 @@
     />
 
     <!-- This site is optimized with the Yoast SEO plugin v20.1 - https://yoast.com/wordpress/plugins/seo/ -->
-    <title>Home - SevenStar Consultants</title>
+    <title>Local Home - SevenStar Consultants</title>
```

Comment:
Used the "Local" tag to show that the home page is being showcased in a local environment and not online.